public/assets/images/round-4-transparent.svg

```diff
@@ -0,0 +1,98 @@
+<svg width="124" height="124" viewBox="0 0 124 124" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
lol I feel like this is a wild enough svg that a png or jpg is more efficient?

Reply:
the png version is too pixelated :/